_IBC/archway-osmosis.json

```diff
@@ -55,6 +55,21 @@
       "tags": {
         "status": "live"
       }
+    },
+    {
+      "chain_1": {
+        "channel_id": "channel-*",
+        "port_id": "wasm.*"
+      },
+      "chain_2": {
+        "channel_id": "channel-*",
```

Comment:
`"channel_id": "*",`

for convention

Reply:
Shall I update this?
What about the port_id? Shall I update that as well?

Reply:
Updated.